mysqlnd_azure.c

```diff
@@ -618,6 +618,7 @@ MYSQLND_METHOD(mysqlnd_azure, connect)(MYSQLND * conn_handle,
 
     if (PASS == (*pconn)->m->local_tx_start(*pconn, this_func)) {
         mysqlnd_options4(conn_handle, MYSQL_OPT_CONNECT_ATTR_ADD, "_client_name", "mysqlnd");
+        mysqlnd_options4(conn_handle, MYSQL_OPT_CONNECT_ATTR_ADD, "_extension_version", MYSQLND_AZURE_VERSION);
```

Comment:
@hzhang87 You mentioned once we need to expose the driver name in Monlogin. Currently the line above for _client_name is already visible in Monlogin as driver name, should we update this value too? Or Monlogin can also properly parse the field _extension_version?

Reply:
![Select * from p_s.session_account_connect_attrs](https://user-images.githubusercontent.com/4216136/81786828-c25d0180-9532-11ea-99c8-7590c5793275.png)
1. _client_name only give "mysqlnd" , _extension_version gives specific version of mysqlnd_azure. Having discussed with @hzhang87 , we need a method to get extension version number from GW side.
2. GW just show what inside a session attr section but do no parse, from my understand.